api/src/main/resources/liquibase.xml

```diff
@@ -588,50 +588,30 @@
 
     <changeSet id="rest_endpoints_accessed_without_authentication_20240507010" author="mujuzi" >
         <preConditions onFail="MARK_RAN">
-            <not>
-                <sqlCheck expectedResult="0">
-                    SELECT count(*) FROM role_privilege WHERE privilege='Get Visits';
-                </sqlCheck>
-            </not>
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM role_privilege WHERE privilege='Get Queue Entries';
+            </sqlCheck>
         </preConditions>
         <comment>Add "Get Queue Entries" privilege to the roles having "Get Visits"</comment>
         <sql>
             INSERT INTO role_privilege (role, privilege)
             SELECT role, 'Get Queue Entries' from role_privilege rp
-            WHERE rp.privilege = 'Get Visits'
-            AND NOT EXISTS (SELECT role, privilege FROM role_privilege rp2 WHERE rp2.privilege='Get Queue Entries'
-            AND rp2.role=rp.role);
+            WHERE rp.privilege = 'Get Visits';
         </sql>
     </changeSet>
 
     <changeSet id="rest_endpoints_accessed_without_authentication_20240507011" author="mujuzi" >
         <preConditions onFail="MARK_RAN">
-            <not>
-                <sqlCheck expectedResult="0">
-                    SELECT count(*) FROM role_privilege WHERE privilege='Add Visits';
-                </sqlCheck>
-            </not>
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM role_privilege WHERE privilege='Manage Queue Entries';
+            </sqlCheck>
         </preConditions>
-        <comment>Add "Manage Queue Entries" privilege to the roles having "Add Visits"</comment>
+        <comment>Add "Manage Queue Entries" privilege to the roles having "Add Visits" and "Edit Visits"</comment>
         <sql>
             INSERT INTO role_privilege (role, privilege)
             SELECT role, 'Manage Queue Entries' from role_privilege rp
-            WHERE rp.privilege = 'Add Visits'
-            AND NOT EXISTS (SELECT role, privilege FROM role_privilege rp2 WHERE rp2.privilege='Manage Queue Entries'
-            AND rp2.role=rp.role);
-        </sql>
-    </changeSet>
+            WHERE rp.privilege = 'Add Visits';
 
-    <changeSet id="rest_endpoints_accessed_without_authentication_20240507012" author="mujuzi" >
-        <preConditions onFail="MARK_RAN">
-            <not>
-                <sqlCheck expectedResult="0">
-                    SELECT count(*) FROM role_privilege WHERE privilege='Edit Visits';
-                </sqlCheck>
-            </not>
-        </preConditions>
-        <comment>Add "Manage Queue Entries" privilege to the roles having "Edit Visits"</comment>
-        <sql>
             INSERT INTO role_privilege (role, privilege)
             SELECT role, 'Manage Queue Entries' from role_privilege rp
             WHERE rp.privilege = 'Edit Visits'
```

Comment:
If you are going to do this, I think you need to ensure you only insert the distict roles from both, so a single sql statement, something like:

```
INSERT INTO role_privilege (role, privilege)
            SELECT distinct role, 'Manage Queue Entries' from role_privilege rp
            WHERE rp.privilege in ('Add Visits', 'Edit Visits');
```